src/Spain/Mews.Fiscalizations.Spain/Communication/SoapClient.cs

```diff
@@ -42,7 +43,7 @@ internal SoapClient(Uri endpointUri, X509Certificate certificate, TimeSpan httpT
             var response = await GetResponseAsync(xml).ConfigureAwait(continueOnCapturedContext: false);
 
             var soapBody = GetSoapBody(response);
-            return XmlSerializer.Deserialize<TOut>(soapBody.OuterXml, new XmlSerializationData(namespaces: GetSiiNameSpaces()));
+            return XmlSerializer.Deserialize<TOut>(soapBody.OuterXml);
```

Comment:
No need to use the data?

Reply:
No, before, it was in the serilization only.